src/game/g_netmaster.pas

```diff
@@ -35,9 +35,8 @@ interface
   // all timeouts in seconds
   NMASTER_TIMEOUT_CONNECT = 3; // 3 seconds
   NMASTER_TIMEOUT_RECONNECT = 5*60; // 5 minutes
-  //NMASTER_TIMEOUT_RECONNECT = 30; // 5 minutes
-  //NMASTER_FORCE_UPDATE_TIMEOUT = 20;
-  //NMASTER_FORCE_UPDATE_TIMEOUT = 0;
+  // Timeout for client waiting for masters' answers in server browser
+  MASTER_TIMEOUT_CLIENT = 1.25;
```

Comment:
No point in making it real for now. Moreover, it could be stored then as a 32-bit float in executable.
Just pre-multiply it by 1000 and leave a note that this timeout is in msecs.